README.md

```diff
@@ -137,32 +141,96 @@ axios
   .then(response => console.log(response)); // resolves with { another: 'response' } after a 1 second delay
 ```
 
-In this example, if we load our site up with `scenario=failedLogin` in the querystring and then attempt to hit the `login` endpoint, it will fail with a 401. However, the `some-other-endpoint` endpoint will still respond with the response in the `default` scenario as we have not provided one in the `failedLogin` scenario.
+In this example, if we load our site up with `?scenario=failedLogin` in the querystring and then attempt to hit the `login` endpoint, it will fail with a 401. However, the `some-other-endpoint` endpoint will still respond with the response in the `default` scenario as we have not provided one in the `failedLogin` scenario.
+
+## Basic GraphQL mock injection (Fetch)
 
-## Exported interfaces
+Here, we have a React application using `urql` as a GraphQL client. This shows how GraphQL queries work and it can be assumed that if you want to use REST mocks in this application, you can do so as you normally would (see examples above).
+
+In reality, the mock definitions would live at a higher level (like the entrypoint into the application) where they could be injected only if we were in development mode.
+
+```tsx
+import React from 'react';
+import { injectMocks, extractScenarioFromLocation } from 'data-mocks';
+import gql from 'graphql-tag';
+
+const mocks: Scenarios = {
+  default: [
+    {
+      url: /graphql/,
+      method: 'GRAPHQL',
+      operations: [
+        {
+          operationName: 'Query',
+          type: 'query',
+          response: { data: { test: 'test' } }
+        },
+        {
+          operationName: 'Mutation',
+          type: 'mutation',
+          response: { data: { test: 'test' } }
+        }
+      ]
+    }
+  ]
+};
+
+injectMocks(scenarios, extractScenarioFromLocation(window.location));
+
+const Component = () => {
+  const [result] = useQuery({ query: Query }); // result will be data: { test: 'test' }
```

Comment:
Nitpick

Should be: `// result will be { data: { test: 'test' } }` (extra curly brackets)

Reply:
Nice spot. I'll fix that up now :)